Blog Post Finding a Team

anasofiapinho opened the pull request:
This blog post is a draft to be approved. Please check if the links and urls are ok. I will update it as soon as I have the feedback about it.

anasofiapinho committed:
Blog Post Finding a Team

alicetragedy commented on blog/_posts/2016-03-28-4-ways-to-find-a-team.md:
```diff
@@ -0,0 +1,62 @@
+
+![Rails Girls Summer of Code Teams App](/img/blog/2016/RGSoC-Teams-App-Find-a-Team.gif)
+
+When you sign up to [RGSoC Teams App](teams.railsgirlssummerofcode.org), on the top right corner you will find our a link to our [Community](thttps://teams.railsgirlssummerofcode.org/users). There are over 800 people there and even though it is not a seamless process (please bare with us, we are working on it!), here you can filter people by their roles or interests. This means you can look for s ...
```
this link needs to be changed to `http://teams.railsgirlssummerofcode.org` for it to work properly :)

alicetragedy commented on blog/_posts/2016-03-28-4-ways-to-find-a-team.md:
```diff
@@ -0,0 +1,62 @@
+
+![4 Ways to Find a Team for RGSoC 2016](/img/blog/2016/RGSoC-finding-a-team.png)
+
+<div style="text-align:right"><font color="grey"><small><i>Team Rubyherzlein @RGSoC 2015</i></small></font></small></div>
```
can you remove the `div-style` part in the caption? aligning it to the right was an exception for the illustration in one of the last posts, because in that case it looked better, but we usually always stick to left-alignment for captions as well.

alicetragedy commented:
Hey @anasofiapinho, awesome work! :sparkles: 
This is a bit of an "annoying" request, but could you rename all images to have the `Find-a-Team` part of their title at the beginning? So that in the `img/blog/2016` folder they will all be grouped together and we know that they belong to the same post. (Right now it doesn't make much of a difference, but by the end of the year there will be lots of images in there!)
Obviously all links to the images will then need to be updated in the post. thank you! :+1:

anasofiapinho commented:
@alicetragedy, the request you made about the name of the images is not annoying! It makes sense! In fact, when I saved the images in the folder, I thought that the folder was becoming a bit untidy and a person would lose a lot of time to find images there! :)

anasofiapinho committed:
Update of the blog post

* Text changes suggested by @alicetragedy
* Redid the gifs to refine the start and finish of the gifs and improve
the resolution
* Renamed the gifs and images to the name of the blog post
* Updated the link to RGSoC Teams App

Please check if everything is ok!

anasofiapinho commented:
Hey @alicetragedy !
I just made some changes to the post as suggested
- Text changes suggested by @alicetragedy
- New paragraph to ask people to mar conversations as resolved on Google Groups
- Redid the gifs to refine the start and finish of the gifs and improve
  the resolution
- Renamed the gifs and images to the name of the blog post
- Updated the link to RGSoC Teams App

Please check if everything is ok! Thank you!

anasofiapinho committed:
Update Community Link of Teams App

This link was not working as well.

alicetragedy commented:
AWESOME!!! :+1: ready to merge, I can do so later today — and we can tweet/add it to facebook tomorrow. great work @anasofiapinho :sparkling_heart:

anasofiapinho commented:
Thanks @alicetragedy! <3 So happy you liked it! I will take care of the twitter and facebook posts tomorrow after you publish it. Would you mind sending me a message on slack when it's published? Thank you!

alicetragedy commented:
yay! :tada: :confetti_ball:
